contracts/multisig/src/workerset.rs

```diff
@@ -0,0 +1,50 @@
+use std::collections::BTreeSet;
+
+use crate::key::PublicKey;
+use crate::msg::Signer;
+use axelar_wasm_std::Participant;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{HexBinary, StdError, Uint256};
+use sha3::{Digest, Keccak256};
+
+#[cw_serde]
+pub struct WorkerSet {
+    pub signers: BTreeSet<Signer>,
+    pub threshold: Uint256,
+    // for hash uniqueness. The same exact worker set could be in use at two different times,
+    // and we need to be able to distinguish between the two
+    pub created_at: u64,
+}
+
+impl WorkerSet {
+    pub fn new(
+        participants: Vec<(Participant, PublicKey)>,
+        threshold: Uint256,
+        block_height: u64,
+    ) -> Result<Self, StdError> {
```

Comment:
Why do we need to return Result here? What's the possible error?

Reply:
Can't figure that out either

Reply:
The `Result` was already there, but the line `weight: participant.weight.into()` may cause an issue because it's turning `nonempty::Uint256` into a `cosmwasm_std::Uint256`. 

In `nonempty::Uint256`, I dont see any implementations of `Into` except for `#[serde(into = "cosmwasm_std::Uint64")]` which doesn't seem equivalent. I think this means the `.into()` would cause an error for numbers bigger than `Uint64`.

Reply:
`.into()` can never cause an error but `.try_into()`. Get rid of the `Result`.